streams/src/test/java/org/apache/kafka/streams/integration/IQv2IntegrationTest.java

```diff
@@ -178,26 +180,26 @@ public static void after() {
 
     @Test
     public void shouldFailUnknownStore() {
-        final PingQuery query = new PingQuery();
-        final StateQueryRequest<Boolean> request =
+        final KeyQuery<Integer, ValueAndTimestamp<Integer>> query = KeyQuery.withKey(1);
```

Comment:
The query itself doesn't matter for these evaluations, so I just arbitrarily swapped KeyQuery in for PingQuery